tests/console-backend-service/internal/domain/eventing/trigger_test.go

```diff
@@ -0,0 +1,249 @@
+// +build acceptance
+
+package eventing
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/kyma-project/kyma/tests/console-backend-service/internal/resource"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/dynamic"
+
+	tester "github.com/kyma-project/kyma/tests/console-backend-service"
+	"github.com/kyma-project/kyma/tests/console-backend-service/internal/client"
+	"github.com/kyma-project/kyma/tests/console-backend-service/internal/graphql"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTriggerEventQueries(t *testing.T) {
+	c, err := graphql.New()
+	assert.NoError(t, err)
+
+	eventingCli, _, err := client.NewDynamicClientWithConfig()
+	require.NoError(t, err)
+
+	namespace, err := fixNamespace(eventingCli, t.Logf)
```

Comment:
Please also remove namespace if error will appear later in the code -> use `defer()`